linter/rules/label_custom_check.py

```diff
@@ -11,12 +11,16 @@ def applies_to() -> Tuple[str, ...]:
     def run(self, lookml_object, runtime_params: Union[Any, None] = None) -> bool:
         label = lookml_object.get('label')
         if label:
-            words_in_label = label.split(' ')
-            for word in words_in_label:
-                valid = starts_with_capital_or_digit_or_special_char(word)
-                if not valid:
-                    return False
+            # Check if all characters are whitespaces
+            if all(char.isspace() for char in label):
```

Comment:
.isspace() returns True also for new line char, but this is good, so not only whitespaces.

Reply:
True, left it and amended the comment. ✅